### DISCUSSION TITLE
test: Add unit tests to client

### DISCUSSION THREAD

**Discussion by danorlando** [danny-avila/LibreChat#501] [Help Wanted]
This started as a hobby project with minimal developers and has since grown considerably. We still have a very limited set of development resources that are trying to keep up with the rapidly evolving AI industry along with the many feature requests coming in from the community. This has made it very hard for us to go back and get test coverage on the repo. 

This is a good first issue for developers who would like to get involved with the project as it will help familiarize with the code for the purpose of making further contributions.

**Comment by distroaryan** [comment]
Hii, I want to contribute , but this is my first open source contribution can you tell me how can i fix it . I will try my best not to waste your time :)

**Comment by Cfomodz** [comment]
I believe this discussion is ready to be closed :) 🚀
